packages/html/stories/Animation.stories.ts

```
@@ -74,21 +74,21 @@ const Template = ({ label, ...args }) => {
         strokeWidth: 3,
         endArrow: 'block',
         endSize: 2,
-        endFill: 1,
+        endFill: true,
         strokeColor: 'black',
-        rounded: 1,
+        rounded: true,
       },
     });
-    e1.geometry.points = [new Point(230, 50)];
+    e1.geometry && (e1.geometry.points = [new Point(230, 50)]);
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)